package.json

```diff
@@ -23,11 +23,11 @@
     "url": "https://github.com/thoughtbot/bitters/issues"
   },
   "homepage": "http://bitters.bourbon.io",
-  "dependencies": {
-    "gulp": "^3.8.11",
+  "devDependencies": {
+    "bourbon": "^4.2.6",
```

Comment:
Wouldn't `bourbon` still need to go in `dependencies` to be importable in their projects?